.github/workflows/ui-test/sanity.json

```diff
@@ -107,6 +107,12 @@
       "PS_VERSION_END": "9.0.0",
       "PHP_VERSION": "8.1",
       "UPGRADE_CHANNEL":  "local"
+    },
+    {
+      "PS_VERSION_START": "8.1.7",
+      "PS_VERSION_END": "9.1.0",
```

Comment:
@AureRita If you plan to keep this test in the future, I would recommend to rename this destination version as nightly, to avoid modifying the value on each release.